dropzone/dropzone.d.ts

```diff
@@ -45,10 +47,12 @@ interface DropzoneOptions {
 	clickable?: boolean|string|HTMLElement|(string|HTMLElement)[];
 	ignoreHiddenFiles?: boolean;
 	acceptedFiles?: string;
+	renameFilename?: RenameFilenameFunction;
```

Comment:
It's fine, but to get more clear IDE suggestion and make it more readable is better to define it as a function, like this: `renameFilename?(name:string):string`